packages/catalog-realm/solitaire/solitaire.gts

```diff
@@ -43,28 +61,70 @@ class IsolatedTemplate extends Component<typeof Solitaire> {
   @tracked selectedCard: Card | null = null;
   @tracked selectedPile: { type: string; index: number } | null = null;
   @tracked moves = 0;
-  @tracked seconds = 0;
   @tracked isPlaying = false;
   @tracked gameWon = false;
+  @tracked gameState = this.args.model.gameState || 'notStarted';
+  @tracked gameMessage = this.args.model.gameMessage || 'Start a new game';
```

Comment:
Just call a function that pass in the message to show in the modal. We don't have to manage gameMessage state here